src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_3_horizontal_layout.html.twig

```diff
@@ -23,30 +23,35 @@ col-sm-2
 {# Rows #}
 
 {% block form_row -%}
+    {%- set widget_attr = {} -%}
+    {%- if help is not empty -%}
+        {%- set widget_attr = {attr: {'aria-describedby': id ~"_help"}} -%}
+    {%- endif -%}
     <div class="form-group{% if (not compound or force_error|default(false)) and not valid %} has-error{% endif %}">
         {{- form_label(form) -}}
         <div class="{{ block('form_group_class') }}">
-            {{- form_widget(form) -}}
+            {{- form_widget(form, widget_attr) -}}
+            {{- form_help(form) -}}
             {{- form_errors(form) -}}
         </div>
-{##}</div>
+    </div>
```

Comment:
What was the reason for these changes?

Reply:
Oops, were they for something? It's just because when I fix something, I've found a lot of them in the template, and I've think it's an error. I can comeback and re-put it.

Reply:
they were here to control how whitespaces are handled

Reply:
Oh okay, I cancel modifications.